src/cuda/common.cu

```diff
@@ -26,15 +26,18 @@ using std::max;
 // a , f are intermediate results from function onedim_fseries_kernel_precomp()
 // (see cufinufft/contrib/common.cpp for description)
 template<typename T>
-__global__ void fseries_kernel_compute(int nf1, int nf2, int nf3, T *f,
-                                       cuDoubleComplex *a, T *fwkerhalf1, T *fwkerhalf2,
-                                       T *fwkerhalf3, int ns) {
+__global__ void fseries_kernel_compute(int nf1, int nf2, int nf3, T *f, T *a,
```

Comment:
Could you comment that this is the equispaced frequency case, used by type 1 & 2, matching onedim_fseries_kernel in CPU code?   And the other fseries_kernel_compute is arbitrary freqs, used by type 3 only?   Otherwise the overlaoding is confusing. thnx.